tests/Unit/Helpers/Domain/CoordinateMaps/TestMapHelpers.hpp

```diff
@@ -360,16 +360,17 @@ void test_coordinate_map_argument_types(
     }
     {
       INFO("Test inverse Jacobian");
+      Approx custom_approx = Approx::custom().epsilon(1.e-11);
```

Comment:
Hmm I'm not sure changing the approx only here is a good idea, but I'm also not sure how else to avoid some slightly larger differences.